app/account/page.tsx

```diff
@@ -0,0 +1,143 @@
+          >
+            <Flex
+              direction="column"
+              align="center"
+              gap={{ base: '40px', lg: '60px' }}
+            >
+              <Heading
+                as="h1"
+                fontSize={{ base: 'xl', md: '3xl', lg: '4xl' }}
+                fontWeight="bold"
+                w={{ base: '100%', lg: 'auto' }}
+              >
+                Account
+              </Heading>
+              <Flex
+                minW={{ base: '100%', lg: '480px' }}
+                gap={{ base: '20px', lg: '30px' }}
+                padding="18px"
+                borderWidth="1px"
+                direction="column"
+                align="space-between"
+              >
+                <Flex direction="column" gap="20px" mb="30px">
+                  <VStack align="start">
+                    <Heading fontSize="bold">Email</Heading>
+                    <Text>{user?.email}</Text>
+                  </VStack>
+                  <VStack align="start">
+                    <Heading fontSize="bold">Name</Heading>
+                    <Text>{user?.name}</Text>
+                  </VStack>
+                  <VStack align="start">
+                    <Heading fontSize="bold">Image</Heading>
+                    {user?.profile_picture_url ? (
+                      <Image
+                        borderRadius="full"
+                        boxSize="30px"
+                        src={user.profile_picture_url}
+                        alt="Profile Picture"
+                      />
+                    ) : (
+                      <MdAccountCircle size="30px" />
+                    )}
+                  </VStack>
+                  <VStack align="start">
+                    <Heading fontSize="bold">Password</Heading>
+                    <Link
+                      href="/"
+                      hasUnderLine
+                      color="primary.700"
+                      fontSize={{ base: 'md', md: 'lg' }}
+                    >
+                      Change your password
+                    </Link>
+                  </VStack>
+                </Flex>
+                <Flex direction="column" align="center">
+                  <VStack justifyContent="center">
+                    <Link href="/account/edit">
+                      <PrimaryButton
+                        variant="solid"
+                        leftIcon={<CiEdit size="16px" />}
+                      >
+                        Edit Profile
+                      </PrimaryButton>
+                    </Link>
+                  </VStack>
+                </Flex>
+              </Flex>
+            </Flex>
+            <Flex direction="row" justify="center">
+              <AlertButton
+                variant="outline"
+                color="red"
+                _hover={{ bg: 'red.500', color: 'white' }}
+                leftIcon={<FaRegTrashAlt size="14px" />}
+              >
+                Delete your account
+              </AlertButton>
```

Comment:
I modified theme file so you don't have to customize manually. Can you delete `_hove` and `color`? Sorry 🙏 

It should be like this after my PR merged.
https://github.com/tabi-memo/frontend/pull/17/files#diff-a6518bbafda819827a4d3cf132487c5474d738ec6bfee9cc3fa1a1882dc78af1
<img width="94" alt="Screenshot 2023-12-22 at 3 11 37 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/93e05d7c-b8ea-405d-89ae-cdbbee322325">

Reply:
I fixed it.
https://github.com/tabi-memo/frontend/pull/13/commits/326c111e6dddf1039531d53b93446d715d48ea12